torch/_dynamo/variables/base.py

```diff
@@ -636,12 +640,12 @@ def __init__(
                 assert source is not None
 
 
-def raise_type_error_exc(tx: "InstructionTranslator", msg_str: str) -> None:
+def raise_type_error_exc(tx: Any, msg_str: str) -> None:
```

Comment:
Doesn't this always return an exception? Should be NoReturn or Never return type from typing/typing_extensions. That tells mypy/pyrefly it always throws.